docs/docsite/rst/installation_guide/intro_configuration.rst

```diff
@@ -42,7 +42,7 @@ Environmental configuration
 Ansible also allows configuration of settings using environment variables.
 If these environment variables are set, they will override any setting loaded from the configuration file.
 
-You can get a full listing of available environment variables from :doc:`_config`.
+You can get a full listing of available environment variables from :doc:`/reference_appendices/config`.
```

Comment:
We're using refs now. Please add the following as the very first line of the file ../reference_appendices/config.rst:

.. _ansible_config:

[..rest of file...]

...and then change this line to: 

:ref:`ansible_config`

Thanks!